code/game/objects/structures/barricade/handrail.dm

```diff
@@ -40,6 +41,22 @@
 		if(E.icon_path && E.obj_icon_state_path)
 			overlays += image(E.icon_path, icon_state = E.obj_icon_state_path)
 
+/obj/structure/barricade/handrail/Collided(atom/movable/movable)
+	if(istype(movable,/mob/living/carbon/xenomorph/ravager) || istype(movable,/mob/living/carbon/xenomorph/crusher))
+		var/mob/living/carbon/xenomorph/xenomorph = movable
+		if(!xenomorph.stat) //No dead xenos jumpin on the bed~
+			visible_message(SPAN_DANGER("[xenomorph] plows straight through [src]!"))
+			deconstruct(FALSE)
+	else
+		if(ismob(movable) && autoclimb)
+			var/mob/living/climber = movable
+			if(climber.a_intent == INTENT_HARM)
+				var/climbed = do_climb(climber)
+				if(climbed)
+					climber.apply_damage(15,BRUTE)
```

Comment:
This is too much damage, probably doesn't need to always happen (prob 25 or something), should ensure `no_limb_loss` is set, and needs to induce movement delay (really goofy how fast you just vault any rails in sequence).

Req is going to get really bloody with this PR. I highly recommend you consider downloading strongdmm to edit req's railing to not have this behavior either with a var edit or a new subtype.

![the worst vaulter](https://github.com/user-attachments/assets/a6eef130-d777-44f6-afec-0c6e3168b401)

Reply:
15 damage is still too much even with 25 prob. Try something more like 5.